client/post-editor/media-modal/detail/detail-preview-image.jsx

```diff
@@ -15,7 +16,14 @@ export default React.createClass( {
 
 	propTypes: {
 		site: PropTypes.object,
-		item: PropTypes.object.isRequired
+		item: PropTypes.object.isRequired,
+		onLoad: PropTypes.func,
```

Comment:
Unless we want to perform a second get, I don't think we need this pass through anymore